README.md

```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
-provisioned into your Azure subscription at cost:
-
-| Count | Resource                       | Type         | Used For |
-|-------|--------------------------------|--------------|----------|
-| 1 | [Azure Active Directory Application][azure-active-directory] | | User Authentication |
-| 1 | [Azure Container Service][container-service] | | [Kubernetes](https://kubernetes.io/) orchestrator |
-| 4 | [Linux Virtual Machines][virtual-machines] | Standard D1 V2 (1 core, 3.5GB memory) | 1 master VM & 3 agents for hosting microservices with redundancy |
-| 1 | [Azure IoT Hub][iot-hub] | S1 – Basic tier | Device management and communication |
-| 1 | [Azure Cosmos DB][cosmos-db] | Standard | Storing configuration data, and device telemetry like rules, alarms, and messages |
-| 5 | [Azure Storage Accounts][storage-account] | Standard GRS | 4 for VM storage, and 1 for the Stream Analytics checkpoints |
-| 1 | [Web Application][web-application] | | Hosting front-end web application |
-
-> Pricing information for these services can be found
-[here](https://azure.microsoft.com/pricing). Usage amounts and billing details
-for your subscription can be found in the
-[Azure Portal](https://portal.azure.com/).
+## All done? Connect a Physical device!
+By default, the solution once spun up uses simulated devices. You can start adding your
+own devices with the instructions here: 
+* [Connect a physical device](https://docs.microsoft.com/azure/iot-suite/iot-suite-connecting-devices-node)
 
 Architecture Overview
 =====================
 <div align="center">
 <img src="https://user-images.githubusercontent.com/3317135/31914374-44a4be80-b7ff-11e7-86b2-19845ab65d7a.png" width="700" height="auto"/>
 </div>
 
-[Open in draw.io][draw-io-map]
-
-## Microservices & Docker Containers
-Remote Monitoring is the first of our preconfigured solutions to leverage a
-microservices architecture. The solution is available in both .NET and Java.
-Microservices have emerged as a prevalent pattern to achieve scale and flexibility
-(by allowing containers to be scaled individually), without compromising development speed.
-Microservices compartmentalize the code and provide well defined interfaces
-making the solution easier to understand and less monolithic. It also further
-expands options for partners that want to extend our current preconfigured
-solutions to build finished solutions that can be monetized.
-
-**Learn more about Docker Containers**
-* [Install Docker](https://docs.docker.com/engine/installation/)
-* [Common Docker Commands for Remote Monitoring](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#common-docker-commands)
-* [Docker Getting Started](https://docs.docker.com/get-started/)
+[Learn more](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-sample-walkthrough) about the remote monitoring architecture, including the use of microservices and docker containers.
```

Comment:
do we need to capitalize "Remote Monitoring?" or "Docker"

Reply:
Fixed!